core/blockly.js

```diff
@@ -262,7 +262,7 @@ Blockly.onKeyDown_ = function(e) {
   if (deleteBlock && !Blockly.selected.workspace.isFlyout) {
     Blockly.Events.setGroup(true);
     Blockly.hideChaff();
-    Blockly.selected.dispose(/* heal */ true, true);
+    Blockly.selected.dispose(/* heal */ true);
```

Comment:
This is functionally different as the second argument for a rendererd block (BlockSvg) is to animate the disposal